src/style.css

```diff
@@ -1,3 +1,74 @@
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+}
+
 body {
   margin: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: Roboto;
+}
+
+header {
+  display: flex;
+  position: fixed;
+  justify-content: space-between;
+  align-items: center;
+  top: 0;
+  left: 0;
+  height: 60px;
+  width: 100%;
+}
+
+.nav_list {
+  /* stylelint-disable-next-line length-zero-no-unit */
+  height: 60px;
+  /* stylelint-disable-next-line length-zero-no-unit */
+  margin: 0px;
+  display: flex;
+  align-items: center;
+}
+
+.nav_item {
+  list-style: none;
+  padding: 12px;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.nav_link {
+  text-decoration: none;
+  color: black;
+}
+
+.is-active {
+  color: #00ACDC;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  background-color: #00ACDC;
+  height: 4px;
+  width: 37px;
+  top: 30px;
+  left: 0px;
+  border-radius: 8px;
+}
+
+.over:hover {
+  color: #00ACDC;
```

Comment:
There should be a css variable for it:

` CSS Variable is used for a blue color
`

Reply:
And look on this comment to, use variables for values what you will reuse